tests/pages/Pilotage.page.test.tsx

```diff
@@ -88,6 +89,19 @@ describe('Pilotage', () => {
         )
       })
 
+      it('résume les activités', async () => {
+        // Then
+        expect(screen.getByRole('heading', { level: 2 })).toHaveTextContent(
+          'Nouvelles activités'
+        )
+        expect(getByDescriptionTerm('Les actions')).toHaveTextContent(
+          '25 à qualifier'
```

Comment:
```suggestion
          '25 À qualifier'
```
Le test fail sinon à cause de la majuscule